modules/contact-form/grunion-contact-form.php

```diff
@@ -946,11 +958,18 @@ public function personal_data_search_filter( $search ) {
 		 * `CHAR(13)` = `\r`, `CHAR(10)` = `\n`
 		 */
 		if ( preg_match( '/AUTHOR EMAIL\: ([^{\s]+)/', $search, $m ) ) {
+			$option_name = sprintf( '_jetpack_pde_feedback_%s', md5( $m[1] ) );
```

Comment:
This is a little hacky. It might be better to store the `$last_post_id` as a property on the class instance instead of calling `get_option()` here. I think either way works.

Reply:
It does feel a little hacky to make these extra db writes during the procedure. In 99% of the cases, I'm sure the impact would be negligible, but I'd probably go with the class property instead.

Reply:
It has to persist across multiple page loads (each page's worth of feedbacks is deleted in a separate AJAX request), so we're stuck with some sort of DB writes.

The property v. `get_option()` issue here is "just" "aesthetics" (maintainability, grokability, etc.). I don't really like either, but I'm happy with both :)

Thanks for testing!